draft-ietf-rtcweb-jsep.xml

```diff
@@ -1064,10 +1072,11 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             immediately; if increased, additional candidates are pre-gathered;
             if decreased, the now-superfluous candidates are discarded.
             </t>
-            <t>Any changes to the BUNDLE policy take effect immediately, i.e.
+            <t>Any changes to the BUNDLE or RTCP multiplexing policies take effect immediately, i.e.
             any future tracks added to the PeerConnection will have their
-            bundle-only state marked accordingly.
+            bundle-only and RTP/RTCP multiplexing state marked accordingly.
```

Comment:
Well, what I thought it meant in this context was that if we added a new track, we would gather RTCP or not as indicated here. Does that not make sense?